### PR TITLE
update main-screen

### PR DESCRIPTION
**Pull request by sertony**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -1405,21 +1405,18 @@
...
             "activate": "활성화",
-            "add": "추가",
-            "assets": "자산",
-            "assetsBalance": "자산 잔액",
             "buy": "구입",
             "contacts": "연락처",
             "convert": "전환",
-            "deposit": "예금",
             "devers": "데버",
+            "earn": "Earn",
```
Comment (sertony) [inline src/ko.json]: @kkkhesha can you translate, please?

File: src/ko.json [hunk 2]
```diff
@@ -1405,21 +1405,18 @@
...
             "evers": "에버",
-            "invite": "초대하기",
             "receive": "받기",
             "send": "보내기",
             "showAssets": "주소 표시",
             "swap": "바꾸기",
             "tokens": "토큰",
+            "topUp": "Top-up",
```
Comment (sertony) [inline src/ko.json]: @kkkhesha can you translate, please?

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [reply src/ko.json]:
```suggestion
            "earn": "보상받기",
```

File: src/ko.json (hunk 2 above)
Comment (kkkhesha) [reply src/ko.json]:
```suggestion
            "topUp": "잔액충전",
```

**Pull request merged**